core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -430,26 +430,21 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
   }
 
   rejectAndReviewNext(reviewMessage: string): void {
-    if (this.validatorsService.isValidReviewMessage(reviewMessage,
-      /* ShowWarnings= */ true)) {
-      this.resolvingSuggestion = true;
-      this.siteAnalyticsService.registerContributorDashboardRejectSuggestion(
-        'Translation');
-
-      // In case of rejection, the suggestion is not applied, so there is no
-      // commit message. Because there is no commit to make.
-      this.contributionAndReviewService.reviewExplorationSuggestion(
-        this.activeSuggestion.target_id, this.activeSuggestionId,
-        AppConstants.ACTION_REJECT_SUGGESTION,
-        reviewMessage || this.reviewMessage, null,
-        this.resolveSuggestionAndUpdateModal.bind(this),
-        (error) => {
-          this.alertsService.clearWarnings();
-          this.alertsService.addWarning(
-            'There was an error rejecting this suggestion');
-        }
-      );
-    }
+    this.finalCommitMessage = this.generateCommitMessage();
+    this.resolvingSuggestion = false;
+    this.siteAnalyticsService.registerContributorDashboardRejectSuggestion(
+      'Translation');
+
+    this.contributionAndReviewService.reviewExplorationSuggestion(
+      this.activeSuggestion.target_id, this.activeSuggestionId,
+      AppConstants.ACTION_REJECT_SUGGESTION,
+      this.reviewMessage, this.finalCommitMessage,
+      this.resolveSuggestionAndUpdateModal.bind(this),
+      (errorMessage) => {
+        this.alertsService.clearWarnings();
+        this.alertsService.addWarning(
+          'There was an error rejecting this suggestion');
+      });
```

Comment:
Are these changes necessary? It seems to me that just passing the review message in https://github.com/oppia/oppia/blob/70d3d05ffc0f72dcf71cae323339410e8dff77fa/core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.html#L163 would solve this issue.

Reply:
Done